api/src/main/resources/api/service_openapi.yaml

```diff
@@ -444,6 +446,23 @@ components:
       type: string
       enum: ["AND", "OR"]
 
+    UnaryFilter:
+      type: object
+      description: A filter function for operating on a single operand.
+      properties:
+        operands:
+          type: array
```

Comment:
This shouldn't be an array, since unlike the array operators there is only a single operand (as your description above says).

Reply:
Done.